Python/Get_BBC_News/getNews.py

```diff
@@ -0,0 +1,22 @@
+import requests
+
+def NewsFromBBC():
+     # BBC news api 
+    main_url = " https://newsapi.org/v1/articles?source=bbc-news&sortBy=top&apiKey=4dbc17e007ab436fb66416009dfb59a8"
+    open_bbc_page = requests.get(main_url).json()
+    articles = open_bbc_page['articles']
+    results = []
+    urls = []
+    
+    for ar in articles:
```

Comment:
Make use of meaningful variable names. `ar` doesn't seem to be consistent with any naming convention.